app/src/docs/components/Search/index.jsx

```diff
@@ -1,121 +1,208 @@
+    letter-spacing: 0.5px;
+    color: #A3A3A3;
+    font-size: 12px;
+`
+
+const NoResultsText = styled(Translate)`
+    color: rgb(50, 50, 50);
+    font-family: var(--sans);
+    letter-spacing: 0;
+    line-height: 28px;
+    font-size: 16px;
+    margin-bottom: 0.5rem;
+
+    @media (min-width: ${LG}px) {
+        font-size: 24px;
+        margin-bottom: 0;
+    }
+`
+
+const MoreInfoText = styled(Translate)`
+    color: rgb(50, 50, 50);
+    font-family: var(--sans);
+    letter-spacing: 0;
+    font-size: 12px;
+    line-height: 18px;
+
+    br {
+        display: block;
+    }
+
+    @media (min-width: ${LG}px) {
+        line-height: 28px;
+        font-size: 16px;
+
+        br {
+            display: none;
+        }
+    }
+`
+
+const TopWrapper = styled.div`
+    background: white;
+`
 
 type Props = {
+    nav?: Node,
     toggleOverlay: () => void,
 }
 
-const Search = ({ toggleOverlay }: Props) => {
+const UnstyledSearch = ({ toggleOverlay, nav, ...props }: Props) => {
     const [index, store] = useGetIndexStore()
     const [query, setQuery] = useState('')
-    const [overlayVisible, setOverlayVisible] = useState(true)
     const searchResults = useLunr(query, index, store)
 
     const onSearchChange = (searchValue) => {
         setQuery(searchValue)
     }
 
     const resultClick = () => {
-        setOverlayVisible(false)
         toggleOverlay()
     }
 
     const closeOverlay = () => {
-        setOverlayVisible(false)
         toggleOverlay()
     }
 
-    const onKeyDown = useCallback((event) => {
-        if (event.key === 'Escape') {
-            setOverlayVisible(false)
-            toggleOverlay()
-        }
-    }, [toggleOverlay])
-
-    useEffect(() => {
-        window.addEventListener('keydown', onKeyDown)
-        return () => {
-            window.removeEventListener('keydown', onKeyDown)
-        }
-    }, [onKeyDown])
-
-    return (overlayVisible
-        ? (
-            <div
-                className={styles.searchOverlay}
-            >
-                <BodyClass className={overlayVisible && 'overflow-hidden'} />
-                <div className={styles.searchContainer}>
-                    <SearchInput
-                        value={query}
-                        onChange={onSearchChange}
-                        placeholder="Search"
-                        className={styles.docsSearchInput}
-                        hideClearButton
-                        autoFocus
-                    />
-                    <button
-                        type="button"
-                        className={styles.exitButton}
-                        onClick={closeOverlay}
-                    >
-                        <SvgIcon name="cross" />
-                    </button>
-                </div>
-                <div className={styles.searchResults}>
+    return (
+        <div {...props}>
+            <BodyClass className="overflow-hidden" />
+            <TopWrapper>
+                {nav}
```

Comment:
the nav is carried over to the search overlay because in the page it might be scrolled out from view (previously it just scrolled back to top)